src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -27,6 +27,7 @@ export default function ColorPickerControl({
             <button
                 className="bp-ColorPickerControl-button"
                 data-testid="bp-ColorPickerControl-button"
+                onBlur={(): void => setIsColorPickerToggled(false)}
```

Comment:
Does clicking on a color still select it before the palette closes? If so, is line 21 still needed with this change?